psf_proposal_SciCookie_2.md

```diff
@@ -0,0 +1,118 @@
+# Proposal for Python Software Foundation
+
+## Form
+
+- Contact
+
+  - Legal First Name*: Ivan
+
+  - Legal Last Name *: Ogasawara
+
+  - Nickname: xmnlab
+
+  - Email *: ivan.ogasawara@gmail.com
+
+- Address (selection)
+
+  - City *:
+
+  - Country*:
+
+  - State/Province/Region:
+
+  - Region Other:
+
+  - Postal Code:
+
+  - Postal Code Suffix:
+
+  - Phone Number *:
+
+- Organization
+  
+  - Organization Name: Open Science Labs
+
+  - Organization Email: opensciencelabs@gmail.com
+
+  - Twitter Handle: @opensciencelabs/
+
+  - Website: https://opensciencelabs.org/
+
+  - Grant Type *: Development work
+
+  - Grant Title *: OSL SciCookie
+
+
+- FAQ & Grants Program Info *: I have read and understand these pages.
+
+- Date *: 12/13/2023
+
+- Event’s/project’s website *: https://github.com/osl-incubator/scicookie
+
+- Requested Grant Amount (USD) *: 2920
+
+- Grant Amount Breakdown *: The money will be used for development work,
+  maintenance, project management and review.
+
+- Budget *: https://docs.google.com/spreadsheets/d/1WlYZVKB-9x3q2WAf0rhcCuukaEMUs7Xn-sE8BqBay-8/edit#gid=528073274
+
+- Objective *:
+
+  Improve the project template SciCookie, the grant would help us to
+  update the dependencies and some configuration changes when necessary.
+  For example, bugfixes, change the TUI lib to something more reliable,
+  improve the configuration for the dependencies (ruff, pre-commit, etc),
+  unify the build-system configuration and others.
+
+  Also at OSL we want to apply for the PyOpenSci review process, so this
+  grant will be used to review all the evaluation criteria, present the
+  project there and apply the suggestions from the review process. It is
+  very important for us to be able to apply to PyOPenSci as it is, if
+  accepted, will allow us to raise awareness of our package and thus
+  contribute to the development of the Python and open source community.  
+
+  Remember that this cookiecutter allows you to automate the process of
+  creating a Python project using a template that has updated
+  technologies that are considered the most appropriate. The options can
+  be configured according to our needs.
+
+  We are looking for Open science labs to grow and to be known by the
```

Comment:
```suggestion
  We are looking for Open Science Labs to grow and to be known by the
```

also remember to remove "we", please.

Reply:
Es que así estaba en la propuesta anterior :D
Cambiado